client/src/partials/journal/modals/trialBalanceMain.footer.html

```diff
@@ -0,0 +1,22 @@
+
+
+<button
+  id="cancelID"
+  type="button"
+  class="btn btn-default"
+  ng-click="TrialBalanceMainFooterCtrl.cancel()">
+    {{ "FORM.BUTTONS.CANCEL" | translate }}
+</button>
+
+<button
+  ng-if="TrialBalanceMainFooterCtrl.state.current.data.checked"
+  type="button"
+  ng-class="{
+    'btn btn-danger' : TrialBalanceMainFooterCtrl.state.current.data.checkingData.feedBack.hasError,
```

Comment:
The ```feedBack``` object in the state is undefined at the loading that why I tailed the service directly to the controller scope.
So doing in the controller : 

``` vm.feedBack = $state.current.data.checkingData.feedBack;```
 will throw an error

>
vendor.min.js:123 TypeError: Cannot read property 'feedBack' of null